cg/apps/demultiplex/demultiplex_api.py

```diff
@@ -42,6 +43,22 @@ def slurm_quality_of_service(self) -> Literal[SlurmQos.HIGH, SlurmQos.LOW]:
         """Return SLURM quality of service."""
         return SlurmQos.LOW if self.environment == "stage" else SlurmQos.HIGH
 
+    @property
+    def set_run_dir_by_sequencer_type(
+        self,
+        config: dict,
+        sequencer_type: str = None,
+    ) -> Path:
+        """Return run dir for sequencer type.
+        Defaults to novaseq flow cell runs dir."""
+        if sequencer_types[sequencer_type] == Sequencers.HISEQX:
+            return Path(config["demultiplex"]["run_dir"]["hiseqx"])
```

Comment:
We probably should use the enum when accessing the config fields as well (replace "hiseqx" etc)